src/javascript/SelectorTypes/SelectorTypes.js

```diff
@@ -87,4 +87,12 @@ export const resolveSelectorType = ({selectorType, selectorOptions}) => {
         selector.key = selectorType;
         return selector;
     }
+
+    if (selectorType) {
+        console.warn(`No renderer component for ${selectorType} selectorType`);
+    } else {
+        console.error(`Field ${displayName} has no selectorType !`, field);
```

Comment:
The `field` used in here is missing the `selectorOptions` (`selectorType` is undefined and `displayName` in the previous sentence), we can add it to the log.

Reply:
ok